tensorflow/training/docker/2.2.0/py3/Dockerfile.cpu

```diff
@@ -28,13 +28,13 @@ ARG PIP=pip3
 ARG PYTHON_VERSION=3.7.7
 ARG OPENSSL_VERSION=1.1.1g
 
-ARG TF_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/tensorflow/r2.2_aws/20200801-180340/cpu/py37/tensorflow_cpu-2.2.0-cp37-cp37m-manylinux2010_x86_64.whl
+ARG TF_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/tensorflow/r2.2_aws/20200810-220210/cpu/py37/tensorflow_cpu-2.2.0-cp37-cp37m-manylinux2010_x86_64.whl
 
 ARG ESTIMATOR_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/estimator/r2.2_aws/20200801-180340/tensorflow_estimator-2.2.0-py2.py3-none-any.whl
```

Comment:
Does ESTIMATOR_URL not need to be modified alongside TF_URL?

Reply:
There are no changes to the estimator binary that are necessary for the functioning of smdebug